src/modules/confirmation/components/ConfirmationSwapTransactionDetails.vue

```diff
@@ -0,0 +1,162 @@
+      </v-col>
+    </v-row>
+    <!--
+      =====================================================================================
+        Values
+      =====================================================================================
+    -->
+    <v-row class="position--relative" justify="space-around">
+      <v-col cols="6" class="text-center value-container">
+        You Swap <br />
+        <img :src="fromImg" height="30px" /> <br />
+        {{ fromVal }} {{ fromType }}
+      </v-col>
+      <v-col cols="6" class="text-center value-container">
+        You will get <br />
+        <img :src="toImg" height="30px" /> <br />
+        {{ toVal }} {{ toType }}
+      </v-col>
+      <div class="icon d-flex align-center">
+        <v-icon> mdi-arrow-right </v-icon>
+      </div>
+    </v-row>
+    <!--
+      =====================================================================================
+        Summary
+      =====================================================================================
+    -->
+    <v-row class="my-4">
+      <v-col cols="6" class="text-left">
+        Provider <br />
+        Exchange rate <br />
+        Transaction fee
+      </v-col>
+      <v-col cols="6" class="text-right">
+        <img :src="provider.exchangeInfo.img" height="25px" /> <br />
+        1 {{ fromType }} = {{ toFixed(provider.rate) }} {{ toType }} <br />
+        <span class="capitalize">{{ gasPriceType }}</span>
+        {{ convertedFees }} {{ txFeeUSD }}
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script>
+import BigNumber from 'bignumber.js';
+import { mapState } from 'vuex';
+import { fromWei } from 'web3-utils';
+export default {
+  props: {
+    provider: {
+      type: Object,
+      default: () => {
+        return {
+          exchangeInfo: {
+            img: ''
+          },
+          amount: '0',
+          rate: '0'
+        };
+      }
+    },
+    gasPriceType: {
+      type: String,
+      default: ''
+    },
+    fromVal: {
+      type: String,
+      default: '0'
+    },
+    toVal: {
+      type: String,
+      default: '0'
+    },
+    to: {
+      type: String,
+      default: ''
+    },
+    from: {
+      type: String,
+      default: ''
+    },
+    fromImg: {
+      type: String,
+      default: ''
+    },
+    toImg: {
+      type: String,
+      default: ''
+    },
+    fromType: {
+      type: String,
+      default: ''
+    },
+    toType: {
+      type: String,
+      default: ''
+    },
+    show: {
+      type: Boolean,
+      default: false
+    },
+    txFee: {
+      type: String,
+      default: '0'
+    },
+    close: {
+      type: Function,
+      default: () => {}
+    },
+    send: {
+      type: Function,
+      default: () => {}
+    }
+  },
+  computed: {
+    ...mapState('external', ['ETHUSDValue']),
+    convertedFees() {
+      return fromWei(this.txFee);
+    },
+    txFeeUSD() {
+      return `$ ${BigNumber(this.convertedFees)
+        .times(this.ETHUSDValue.value)
```

Comment:
not sure if you want to use fiatValue here incase there are different currencies